zkevm-circuits/src/evm_circuit/table.rs

```diff
@@ -311,13 +313,17 @@ impl<F: Field> Lookup<F> {
                 field_tag,
                 index,
                 value,
-            } => vec![id.clone(), field_tag.clone(), index.clone(), value.clone()],
+            } => [
+                vec![id.clone(), field_tag.clone(), index.clone()],
+                value.limbs.to_vec().clone(),
+            ]
+            .concat(),
```

Comment:
Another option for this would be:
```rust
vec![id.clone(), field_tag.clone(), index.clone(), value.lo().clone(), value.hi().clone()]
```
What do you think about this form?

Reply:
I like `value.lo().clone(), value.hi().clone()` style, just I was thinking if using `lo(), hi()` here then once change to different field we need to modify this part. Although it just small change though.

Reply:
@ed255 I realized there are `table.annotations()` which still hardcode column to annotation string. So my methodology doenst really help to generalized. I adapt your method 👍